modules/drivers/vertica/src/metabase/driver/vertica.clj

```diff
@@ -22,9 +22,6 @@
 
 (driver/register! :vertica, :parent #{:sql-jdbc ::legacy/use-legacy-classes-for-read-and-set})
 
-(defmethod driver/supports? [:vertica :percentile-aggregations] [_ _] false)
```

Comment:
Does Vertica support percentile aggregations? @sbelak would we be breaking anything by removing this?

Reply:
Supposedly yes, but I couldn’t get it to working following the example in their documentation. I think it's best to leave it off for now, and I'll open a new issue for it.